lib/trento/application/read_models/check_result_read_model.ex

```diff
@@ -17,9 +15,8 @@ defmodule Trento.CheckResultReadModel do
     field :cluster_id, Ecto.UUID, primary_key: true
     field :host_id, Ecto.UUID, primary_key: true
     field :check_id, :string, primary_key: true
-    field :result, Ecto.Enum, values: [:passing, :warning, :critical, :running]
+    field :result, Ecto.Enum, values: [:passing, :warning, :critical, :unknown]
```

Comment:
Other small comment. From the runner, we pass the `skipped` result as well to show which were the skipped checks.

Reply:
noted, will come in follow up PRs